runtime/src/accounts_db.rs

```diff
@@ -2392,7 +2389,7 @@ impl AccountsDb {
         paths: Vec<PathBuf>,
         cluster_type: &ClusterType,
         account_indexes: AccountSecondaryIndexes,
-        caching_enabled: bool,
+        _caching_enabled: bool,
```

Comment:
will get removed in a later pr to limit fanout noise and focus on behavior changes.